_posts/2022-09-06-deepchecks-and-zenml.md

```diff
@@ -270,17 +272,6 @@ from zenml.steps import Output, step
 
 from deepchecks.tabular.datasets.classification import iris
 
-# import the Deepchecks data and model validation step utilities
-from zenml.integrations.deepchecks.steps import (
-    DeepchecksDataIntegrityCheckStepConfig,
-    deepchecks_data_integrity_check_step,
-    DeepchecksDataDriftCheckStepConfig,
-    deepchecks_data_drift_check_step,
-    DeepchecksModelValidationCheckStepConfig,
-    deepchecks_model_validation_check_step,
-    DeepchecksModelDriftCheckStepConfig,
-    deepchecks_model_drift_check_step,
-)
```

Comment:
These imports seem to be required. Maybe you removed them by mistake ?

Reply:
Yes, imports are required! actually, i was trying to split the code into different chunks rather than one long one but decided against it in the end and seems that forgot to revert this

Reply:
Yes, imports are required! actually, i was trying to split the code into different chunks rather than one long one but decided against it in the end and seems that forgot to revert this